<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="StationList_Table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddToList_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearList_Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseWindow_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportCSV_Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export CSV</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
